src/security/DigestAlgorithm.h

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 1996-2018 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#ifndef SQUID__SRC_SECURITY_DIGESTALGORITHM_H
+#define SQUID__SRC_SECURITY_DIGESTALGORITHM_H
+
+namespace Security
+{
+
+#if USE_OPENSSL
+typedef EVP_MD const * DigestAlgorithm;
```

Comment:
There are no headers required to define those. The TestHeaders unit checks enforce that property and build just fine. I suspect that since this is a typedef of a raw-pointer type and not actually used, the compiler does not need to know the pointed-to objects exact details to define these types.

Reply:
Update: the changes to header includes have found the needed ones. So this is done.